modules/monitoring-choosing-a-metrics-collection-profile.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="choosing-a-metrics-collection-profile_{context}"]
+= Choosing a metrics collection profile
+
+To choose a metrics collection profile for core {product-title} monitoring components, edit the `cluster-monitoring-config` `ConfigMap` object.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have enabled Technology Preview features by using the `FeatureGate` custom resource (CR).
+
+[WARNING]
+====
+Saving changes to a monitoring config map might restart monitoring processes and redeploy the pods and other resources in the related project.
+The running monitoring processes in that project might also restart.
+====
+
+.Procedure
+
+
+. Edit the `ConfigMap` object:
```

Comment:
remove this line, the next line is edit the `cluster-monitoring-config` `ConfigMap`

Reply:
fixed